install/configuring_inventory_file.adoc

```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
 a|This variable configures additional `docker` options in
 *_/etc/sysconfig/docker_*, such as options used in
 xref:host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-It is recommended to use `json-file`.
+We recommend that you use `json-file`.
```

Comment:
The style guide says not to use "Do not use to refer to a company (for example, “We recommend...”)."

It would be better to explain why you need to use `json-file`, otherwise revert the change.

Reply:
@kalexand-rh This is yet another issue we should get before the team. There are several constructs around recommended/not recommended looking in Atom.

Reply:
It is also something we are talking about in the Doc Conventions initiative -- looking at whether we should change that. I'll be sure to bring it up in the next meeting again. Thanks!